CONTRIBUTING.md

```diff
@@ -25,31 +25,62 @@ By contributing to Cube Dev, Inc., You accept and agree to the terms and conditi
 Cube.js works with Node.js 8+ and uses yarn as a package manager.
 
 ## Development Workflow
+
+### Cube.js Docker
+
+Cube.js offers two different types of docker's image:
+
+- Stable (building from published release on npm)
+- Dev (building from source files, needed to test unpublished changes)
+
+For more information, take a look at [Docker Development Guide](./packages/cubejs-docker/DEVELOPMENT.md).
+
+#### Stable Docker Release
+
+1. After cloning Cube.js repository run `$ yarn` in `packages/cubejs-docker` to install dependencies.
```

Comment:
Can we remove the `$` from the beginning of all the inline commands, please?

Reply:
I am using similar style across file, it was not added by me. 

cC @keydunov Is it ok to remove `$`?

Thanks

Reply:
Let's keep `$` for now as we're using it everywhere